lib/datasource/index.spec.ts

```diff
@@ -116,10 +116,9 @@ describe(getName(), () => {
       datasource: datasourceNpm.id,
       depName: 'react-native',
     });
-    // FIXME: explicit assert condition
-    expect(res).toMatchSnapshot();
-    expect(res.changelogUrl).toBeDefined();
-    expect(res.sourceUrl).toBeDefined();
+    expect(res).toMatchSnapshot({
+      changelogUrl: expect.any(String),
```

Comment:
Why not match real values?

Reply:
I wanted to make an accent on that `changelogUrl` exists and is string, but didn't expect it writes `Any<string>` instead of real value in the snapshot file. Thus, I'm reverting it to real value matching.

Reply:
👍😅